src/palindrom.js

```diff
@@ -780,6 +781,16 @@ var Palindrom = (function() {
         );
       }
     });
+    /* when patches are applied, and their `value`s are objects, make sure to pass the proxified version */
```

Comment:
Sorry, but I don't see a reason to modify the original sequence. 
I would even say it SHOULD remain intact as this is what remote requested.